include/CL/sycl/handler.hpp

```diff
@@ -31,6 +31,8 @@
 namespace cl {
 namespace sycl {
```

Comment:
Not relevant to the commit but if it's possible might as well change it to the newer nested namespaces definition e.g. `namespace cl::sycl` as was done for task.hpp, could also do it for queue.hpp if it requires it so we gradually move towards the new style.